src/openrct2/world/Sprite.h

```diff
@@ -194,9 +194,9 @@ enum
 rct_sprite* try_get_sprite(size_t spriteIndex);
 rct_sprite* get_sprite(size_t sprite_idx);
 
-extern uint16_t gSpriteListHead[6];
-extern uint16_t gSpriteListCount[6];
-extern uint16_t gSpriteSpatialIndex[0x10001];
+extern uint16_t gSpriteListHead[SPRITE_LIST_COUNT];
+extern uint16_t gSpriteListCount[SPRITE_LIST_COUNT];
+extern uint16_t gSpriteSpatialIndex[MAX_SPRITES + 1];
```

Comment:
This is incorrect the spatial index should still be 0x10001

Reply:
Nah, it should be `MAX_MAP_SIZE * MAX_MAP_SIZE + `